src/components/flat-table/flat-table-sizes.stories.mdx

```diff
@@ -0,0 +1,59 @@
+import { Meta, Props, Preview, Story } from '@storybook/addon-docs/blocks';
+import Box from "../box"
+import { 
+  FlatTable, 
+  FlatTableHead, 
+  FlatTableBody, 
+  FlatTableRow, 
+  FlatTableHeader,
+  FlatTableCell,
+} from '.';
+
+<Meta title="Design System/Flat Table/Sizes" parameters={{ info: { disable: true }}} />
+
+## Table Cell Sizes
+
+<Preview>
+  <Story name="compact">
```

Comment:
Story name should not be compact. Should be something like "sizes" as it shows all the sizes. Or might be worth removing the sizes folder and placing this story alongside the others in the Design System/Flat Table folder

Reply:
changed to `sizes` and moved to common folder